dev-manuals/test-plans/template.md

```diff
@@ -1,32 +1,31 @@
-# Name (`Test Case ID`)
+# $CATEGORY: $ACTION (`$TESTCASE_ID im Format "u.$(kebab-case $CATEGORY)-$NUMBER (from 1)"`)
 
-Test Case name pattern: u.XY(service identifier)-Z (sequential number, start at 1).  
-Version: vx.x, date of creation, description of change  
-Author: Author of the use case description  
-Tester: Use case tester
+Version: `v$X.$Y with $X >= 1, $Y >= 0`, `$NOW in YYYY-mm-dd`, `$CHANGE_DESCRIPTION like a git commit` \
+Author: Authors of this file, comma-separated
+(Delete this line in test cases!) Filename: `$(kebab-case u-$CATEGORY-$NUMBER-$ACTION).md`
 
 ## Description
 
-Maximum 3 concise sentences about the realized function.
+Maximum of **3 concise** sentences about the realized function.
 
 ## Precondition
 
-Precondition for starting the use case
+Precondition for starting the use case.
 
 ## Postcondition
 
-Postcondition after the use case has ended
+Postcondition after the use case has ended.
 
 ## Typical procedure
 
-Typical procedure of the use case. Description
-by a numbering of the executed steps.
+1. Do `x`
+2. Do `y`
 
 ## Alternative procedures
 
-Alternative procedures existing to the typical procedure
-Description by numbering of the executed
-alternative steps
+Do not show errors here, only intended alternatives in format
+1.1. `condition` - do `x2` \
+2.1. `condition 2` - do `y2`
```

Comment:
```suggestion
*Do not show errors here, only intended alternatives in format* \
1.1. `condition` - do `x2` \
2.1. `condition 2` - do `y2`
```
make the comment recursive or something that it shows it is a comment and not part of the procedure

Reply:
```suggestion
<!-- Do not show errors here, only intended alternatives in format -->
1.1. `condition` - do `x2` \
2.1. `condition 2` - do `y2`
```